### DISCUSSION TITLE
How to get the contributors of the repository?

### DISCUSSION THREAD

**Discussion by A-kirami** [Mergifyio/mergify#5042] [Q&A]
When they start sending their first contributions, I want to welcome them.
But I didn't find the contributor attribute in Mergify.
How should I get current contributors, or how should I distinguish between first time contributors?

**Comment by jd** [comment]
I imagine you mean the PR author?
I'm not sure if this is exposed through GitHub API. If it is we might add this in a condition.

**Reply by A-kirami** [reply]
Yes, I want to know how to judge that the PR author is not among the repository contributors.
